conanfile_rtc.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile
+
+
+class PROJConan(ConanFile):
+    name = "PROJ"
+    version = "v8.1.0"
+    url = "https://github.com/Esri/PROJ/tree/runtimecore"
+    license = "https://github.com/OSGeo/PROJ/blob/master/COPYING"
+    description = "PROJ"
+
+    # RTC specific triple
+    settings = "platform_architecture_target"
+
+    def package(self):
+        base = self.source_folder + "src/"
+        relative = "3rdparty/PROJ/"
+
+        # headers
+        self.copy("*.h*", src=base, dst=relative)
```

Comment:
Are excludes needed here too like the lua file has?

Reply:
Yes, will fix.